scripts/prettier.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+
+// Based on similar script in React
+// https://github.com/facebook/react/blob/b87aabdfe1b7461e7331abb3601d9e6bb27544bc/scripts/prettier/index.js
+
+import glob from 'glob';
+import prettier from 'prettier';
+import fs from 'fs';
+import os from 'os';
+import listChangedFiles from './listChangedFiles';
+
+const prettierConfigPath = require.resolve('../prettier.config');
```

Comment:
import?

Reply:
This resolves the path the alternative is: `path.join(__dirname, '../prettier.config.js')`